app/helpers/claims_helper.rb

```diff
@@ -46,7 +46,19 @@ def fee_shared_headings(claim, scope)
       page_header: t('page_header', scope: scope),
       page_hint: t('page_hint', scope: scope)
     }.tap do |headings|
-      headings[:page_notice] = t('unused_materials_notice', scope: scope) if display_unused_materials_notice_for(claim)
+      headings[:page_notice] = t('unused_materials.notice', scope: scope) if display_unused_materials_notice_for(claim)
+    end
+  end
+
+  def misc_fees_locals(claim, args = {})
```

Comment:
🤔 not too sure about this. It extracts logic from the view which is nice but: 
1. this partial is small and specific anyway and this add adds a layer of indirection
2. this helper is specific to misc fee **summary** but it exists in claims helper so could be confused with generic misc fees.

Reply:
options would seem to be 
1. put conditions in partial
2. name change to indicate summary
3. perhaps we need a new claim_summary_helper generally